pkg/watcher/webhook_chart.go

```diff
@@ -75,7 +76,7 @@ func resolveKcpAddr(kcpConfig *rest.Config, managerConfig *SkrWebhookManagerConf
 	externalIP := loadBalancerService.Status.LoadBalancer.Ingress[0].IP
 	var port int32
 	for _, loadBalancerPort := range loadBalancerService.Spec.Ports {
-		if loadBalancerPort.Name == "http2" {
+		if loadBalancerPort.Name == managerConfig.ListenerGatewayPortName {
```

Comment:
The `IngressServiceName` and `IstioSystemNs` better also make them configurable as flags, but not hardcoded.